tests/test_collector.py

```diff
@@ -23,8 +23,13 @@ def fetchMetricsSuccessFromPgBouncer17Mock(conn, query):
         ]
     elif query == "SHOW DATABASES":
         return [
-            {"name": "test","database": "test", "pool_size": 50, "reserve_pool": 10, "current_connections": 30 },
-            {"name": "prod","database": "prod", "pool_size": 90, "reserve_pool": 20, "current_connections": 75 }
+            {"name": "test","database": "test", "pool_size": 50, "reserve_pool": 10, "current_connections": 30, "max_connections": 0},
+            {"name": "prod","database": "prod", "pool_size": 90, "reserve_pool": 20, "current_connections": 75, "max_connections": 5}
+        ]
+    elif query == "SHOW CONFIG":
+        return [
+            {"key": "max_client_conn","value": 500, "changeable": "yes"},
+            {"key": "max_user_connections","value": 0, "changeable": "yes"}
```

Comment:
Please, fix the style of these 2 lines so it's consistent:

```python
            {"key": "max_client_conn",      "value": 500, "changeable": "yes"},
            {"key": "max_user_connections", "value": 0,   "changeable": "yes"}
```

Reply:
It should be ok now with the latest commit